src/components/sbb-toggle-check/sbb-toggle-check.scss

```diff
@@ -0,0 +1,92 @@
+.toggle-check {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  color: var(--toggle-check-color);
+  @include text-s--regular;
+}
+
+.toggle-check__slider--after {
+  margin-inline-end: var(--sbb-spacing-fixed-3x);
+}
+
+.toggle-check__slider--before {
+  margin-inline-start: var(--sbb-spacing-fixed-3x);
+}
+
+input[type='checkbox'] {
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  margin: 0;
+  width: 100%;
+  height: 100%;
+  cursor: inherit;
+  z-index: -1;
+}
+
+.toggle-check__circle {
+  @include shadow-level-5-hard;
+
+  content: '';
+  position: absolute;
+  top: 50%;
+  width: var(--toggle-check-circle-diameter);
+  height: var(--toggle-check-circle-diameter);
+  border-radius: #{px-to-rem-build(24)};
+  background-color: var(--toggle-check-circle-background);
+  will-change: transform;
+  transform: translate(var(--toggle-check-circle-offset), -50%);
+  transition: var(--sbb-animation-duration-6x);
+
+  .toggle-check--checked & {
+    transform: translate(calc(100% - var(--toggle-check-circle-offset)), -50%);
+  }
+}
+
+::slotted([slot='icon']) {
+  fill: var(--toggle-check-fill);
+}
+
+.toggle-check__slider {
+  position: relative;
+  cursor: var(--toggle-check-cursor);
+  display: inline-block;
+  overflow: hidden;
+  min-width: var(--toggle-check-width);
+  height: var(--toggle-check-height);
+  border-radius: var(--toggle-check-height);
+  background-color: var(--toggle-check-background);
+
+  .toggle-check--checked & {
+    background-color: var(--sbb-color-red-default);
+  }
+
+  input[type='checkbox']:focus-visible + & {
+    @include focusOutline;
+  }
+}
+
+:host {
```

Comment:
Place :host at the top

Reply:
the pre commit hook forced me to do this. I guess its because of the &.toggle-check--disabled